### ISSUE TITLE
Album titles containing quotation

### ISSUE THREAD

**Issue by CHJ85** [andyjsmith/SmugMug-Downloader#15]
Hi there.
How do I download albums that contains quote or double quotations in em.
Thanks.

**Comment by andyjsmith** [comment]
That's the most practical way to do it. Since it's entered via command line, you'll have to escape it somehow, like with a backslash.

**Comment by CHJ85** [comment]
Right. But would it be possible to add a URL flag and let the script extract user and album from that?

**Comment by andyjsmith** [comment]
Probably, I just don't have the time to work on this old project right now. You're welcome to implement it yourself and submit a PR.
